src/libs/ReportUtils.js

```diff
@@ -2258,6 +2258,16 @@ function getParentReport(report) {
     return lodashGet(allReports, `${ONYXKEYS.COLLECTION.REPORT}${report.parentReportID}`, {});
 }
 
+/**
+ * Return true if the composer should be hidden
+ * @param {Object} report
+ * @param {Object} errors
```

Comment:
Can we be more specific what kind of `errors` we're dealing with here? I believe the errors that could be passed here are technically only "addWorkspaceRoomOrChatErrors" - so maybe we can just call this `reportErrors` to be safe?